### PR TITLE
Update post_workshop_email.txt

### PR DESCRIPTION
**Pull request by orchid00**
change subject to match https://github.com/swcarpentry/board/blob/master/subcommittees/mentoring/files/pre_workshop_email.txt

### CODE REVIEW

File: subcommittees/mentoring/files/post_workshop_email.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Subject: Data & Software Carpentry Workshop Debriefing
+Subject: Software and Carpentry Workshop Debriefing
```
Comment (mkuzak) [inline subcommittees/mentoring/files/post_workshop_email.txt]: should be Software and Data Carpentry

Comment (orchid00) [reply subcommittees/mentoring/files/post_workshop_email.txt]: yes sorry, typo

**Comment by ChristinaLK** [comment]
@orchid00 can you resubmit in `carpentries/mentoring`?  This folder should be going away soon -- the PR just hasn't been merged yet.

**Comment by mkuzak** [comment]
@ChristinaLK good point I missed that thise is swcarpentry organisation

**Comment by orchid00** [comment]
@ChristinaLK can you send me a link, I don't know where is that

**Comment by mkuzak** [comment]
@orchid00 the mentoring repository is now here: https://github.com/carpentries/mentoring
and the file you edited is: https://github.com/carpentries/mentoring/blob/master/files/post_workshop_email.txt

**Comment by orchid00** [comment]
Thanks @mkuzak I have sent a PR to that repo

**Comment by mkuzak** [comment]
hi @orchid00, thanks. I cannot find the PR you're mentioning.

**Pull request closed without merging**